hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/policies/amrmproxy/LocalityMulticastAMRMProxyPolicy.java

```diff
@@ -232,6 +238,19 @@ public void notifyOfResponse(SubClusterId subClusterId,
       // Handle "node" requests
       try {
         targetId = resolver.getSubClusterForNode(rr.getResourceName());
+
+        // If needed, re-reroute node requests base on SC load
+        // Read from config every time so that it is SCDable
```

Comment:
Replace SCDable

Reply:
BTW, wouldn't this be a little expensive for every time to check the configuration?
Do we need to reload it?

Reply:
There is this JIRA which propses to dynamical load configuration https://issues.apache.org/jira/browse/YARN-10216.
Adding it will have with dynamic loading.